.github/workflows/windows.yml

```diff
@@ -95,10 +95,93 @@ jobs:
         with:
           name: release-double-zipped
           path: build/release/release.zip
+
+  smoke-test-windows:
+    runs-on: windows-2022
+    needs: build-windows
+    steps:
+      - uses: brechtm/setup-scoop@v2
+        with:
+          scoop_update: 'false'
+      - name: Install Dependencies
+        shell: bash
+        run: |
+          scoop install binaryen
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
Are you sure this is necessary? I believe TinyGo ships with binaryen (wasm-opt) already included.
The only thing that's not included is the `Makefile`, which you can get with a checkout without submodules (submodules slow down the checkout a lot).

Reply:
Doesn't this line in the `Makefile` indicate that the default for Windows is to use the system `wasm-opt` meaning I still need the scoop installed version?

https://github.com/tinygo-org/tinygo/blob/dev/Makefile#L132

Makes sense that I can skip the submodules in the checkout.

Reply:
Hmm, I think you're right. I don't remember why this is necessary, would be nice to be able to build wasm-opt on Windows.